klasses/bootcamp_admissions_client.py

```diff
@@ -76,6 +77,19 @@ def fetch_legacy_admissions(user_email):
         return {}
 
 
+def fetch_fluidreview_klass_keys(fluid_user_id):
+    """
+    Collect all the unique award ids (== klass ids) from WebhookRequests by user email
+
+    Args:
+        email(str): A user's email address
+
+    Returns:
+        list: FluidReview award_ids for a user
+    """
+    return list(set(WebhookRequest.objects.filter(user_id=fluid_user_id).values_list('award_id', flat=True)))
```

Comment:
Can you use [https://docs.djangoproject.com/en/1.11/ref/models/querysets/#django.db.models.query.QuerySet.distinct](distinct) instead of using `set` here to remove duplicates?

Reply:
✅